bot/code_review_bot/report/phabricator.py

```diff
@@ -70,25 +78,39 @@ def publish(self, issues, revision, task_failures):
             # Always publish comment summarizing issues
             self.publish_comment(revision, issues_only, patches, task_failures)
 
-            # Also publish build issues
-            if self.publish_build_errors:
-                self.publish_harbormaster_build_errors(revision, build_errors)
+            # Publish all errors outside of the patch as lint issues
```

Comment:
Maybe we should publish all errors as lint issues, no matter if they are outside or inside the patch?

Reply:
I would start with only extra errors as lint issues, see how the dev react, then upgrade to that later on.

Reply:
OK, we can use #25 to track that.

Reply:
https://github.com/mozilla/code-review/issues/25#issuecomment-572570606